apps/pie-docs/src/_11ty/shortcodes/card.js

```diff
@@ -0,0 +1,46 @@
+const pieIconsSvg = require('../filters/pieIconsSvg');
+
+/**
+ * A Card HTML component – takes an array of list items and turns them into a list of cards
+ * @param {string[]} items - An array of card items
+ * @returns {string}
+ */
```

Comment:
the naming is quite misleading here. the name (Card) indicates an atomic component that should only render a card regardless of its context or quantity, but instead, it accepts items and renders multiple cards while its name indicates singularity.

I think ideally we should have a shortcut called card, which only builds a card and is not aware of its surroundings (don't add margins, no layout-specific styles, etc ) so we can drop it anywhere standalone, and then create another shortcut called CardList / CardsGrid / CardsWrapper which adds the layout styles [here](https://github.com/justeattakeaway/pie/pull/271/files#diff-469e20b58f0ff869d00a0e07549f804ed6704128b6816fd78b9d233f9180df79R37-R43).

this way we separate the logic of the card and its container and allow us to reuse it with different layouts like the one we have on the home page, wdyt?

Reply:
I think that would be ideal @raoufswe but unfortunately I don't think you can reference a shortcode from another shortcode. It's a known limitation in Eleventy and why, when 2.0. comes out, it would be cool if we can leverage `webc` instead of shortcodes, or just use our own web component library in the future.

I think given the time constraints, I'd opt for whatever is the simplest approach here to get us to the deadline and then Faye can plan any refactors we'd like to achieve in the next initiative, which she will be leading. 

I appreciate this comes off as kicking the can down the road, but we should bear in mind this is not a component library and I don't think we need to be as stringent for the docs site components as we would for Snacks or Fozzie.

Happy to discuss further though, and it's not just my call :)